CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix race condition while parsing derived fields from search definition ([14445](https://github.com/opensearch-project/OpenSearch/pull/14445))
 - Add allowlist setting for ingest-common and search-pipeline-common processors ([#14439](https://github.com/opensearch-project/OpenSearch/issues/14439))
 - Create SystemIndexRegistry with helper method matchesSystemIndex ([#14415](https://github.com/opensearch-project/OpenSearch/pull/14415))
+- Change default value of batch size parameter of bulk API to Integer.MAX_VALUE ([#14725](https://github.com/opensearch-project/OpenSearch/pull/14725))
```

Comment:
Let's change the entry to something like "Deprecate batch_size parameter on _bulk API" and move this to the "Deprecated" section.